### DISCUSSION TITLE
opentelemetry-java agent separated apm metrics to different index

### DISCUSSION THREAD

**Discussion by CenkuSands** [open-telemetry/opentelemetry-java-instrumentation#10962] [Q&A]
We have 2 types of applications that build by java or dotnet, for java, we are using opentelemetry-java agent (v1.28.0) to send the apm metrics, logs & traces to elasticsearch, and my elasticsearch integration is go through elastic-agent --> fleet server --> elasticsearch.

My issue is, when dotnet based applications send apm metrics to elastic, all my service send apm metrics to the metrics-apm.internal ingest pipelines and store to the **.ds-metrics-apm.internal-default** index, however, for the java based applications, it send apm metrics to elastic that separated by service name, so instead of all services' apm-metrics goes into .ds-metrics-apm.internal-default*, java based services go to **.ds-metrics-apm.app.app01***, **.ds-metrics-apm.app.app02***, etc.

My question is, from  opentelemetry-java agent prospective, could it be set the ingest pipeline/index/index template goes to **.ds-metrics-apm.internal-default** rather than separated by services name?

**Comment by laurit** [comment]
cc @jackshirazi
